src/common/AppToggle.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <label class="container">
+    <input
+      v-bind="$attrs"
+      class="input"
+      type="checkbox"
+      :checked="modelValue"
+      @change="$emit('update:modelValue', $event.target.checked)"
+    />
+
+    <span class="switch">
+      <span :class="{ leftEnabled: !modelValue, leftDisabled: modelValue }">{{
+        leftText
+      }}</span>
+      <span :class="{ rightDisabled: !modelValue, rightEnabled: modelValue }">{{
+        rightText
+      }}</span>
+    </span>
+  </label>
+</template>
+
+<script>
+export default {
+  name: "Switch",
+  props: {
+    leftText: {
+      type: String,
+      required: true
+    },
+    rightText: {
+      type: String,
+      required: true
+    },
+    modelValue: {
+      type: Boolean,
+      required: true
+    }
+  },
+  emits: ["update:checked"],
+  methods: {
+    update() {
+      this.$emit("update");
+    }
+  }
```

Comment:
Is this method used anywhere? I don't see it anywhere — not sure if I'm missing something.